package.json

```diff
@@ -32,7 +33,7 @@
     "version:major": "npm --no-git-tag-version version major",
     "preversion": "npm run test && npm run check-changelog && npm run check-only-changelog-changed",
     "postversion": "git commit package.json CHANGELOG.md -m \"Version $npm_package_version\" && npm run tag && git push && git push --tags && npm publish --registry=https://registry.npmjs.org/",
-    "prepublish": "in-publish && safe-publish-latest && npm run build || not-in-publish",
+    "prepublish": "npm run build",
```

Comment:
the `in-publish` and `safe-publish-latest` checks are required to gate publishing - we don't actually want to run the build upon installing.